templates/taxi/driver_list.html

```diff
@@ -5,10 +5,17 @@ <h1>Driver list</h1>
   {% if driver_list %}
     <ul>
       {% for driver in driver_list%}
-        <li>
-          <a href="{% url "taxi:driver-detail" pk=driver.id %}">{{ driver.username }}</a>
-          ({{ driver.first_name }} {{ driver.last_name }})
-        </li>
+        {% if driver.id == user.id %}  
```

Comment:
Ensure that the `user` object is available in the template context. If `user` is not passed to the template, this comparison will fail.